src/login/InitResetForm/InitResetForm_sagas.js

```diff
@@ -0,0 +1,27 @@
+import { put, select, call } from "redux-saga/effects";
+import { postRequest, checkStatus, saveData } from "sagas/common";
+import * as actions from "login/InitResetForm/InitResetForm_actions";
+
+
+export function sendEmailRequest(config, data) {
```

Comment:
Do we have this saga (to a different endpoint) in signup? Are they the same? Can they be combined?

Reply:
There is no similar saga in the Email component of the signup app. The signup app 1st collects the email and then asks the user a captcha and shows the ToU, and then there is a saga that sends the 3 pieces of info (email, captcha response and ToU acceptance) to the backend.